public/public/index.html

```diff
@@ -4,9 +4,7 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Work+Sans&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@200;400&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway&family=Work+Sans:wght@200;400&display=swap" rel="stylesheet">
```

Comment:
I think the syntax you're looking for is `family=Raleway+Work+Sans` but honestly if you go back to google fonts and just add multiple to your "font cart" it will give you the correct thing

Reply:
as a matter of fact that's exactly what i did. so the syntax should be correct